packages/web-components/fast-foundation/src/breadcrumb/breadcrumb.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { expect } from "chai";
+import { Breadcrumb, BreadcrumbTemplate as template } from "./index";
+import { fixture } from "../fixture";
+import { customElement, html } from "@microsoft/fast-element";
+
+@customElement({
+    name: "fast-breadcrumb",
+    template,
+})
+class FASTBreadcrumb extends Breadcrumb {}
+
+async function setup() {
+    const { element, connect, disconnect } = await fixture<FASTBreadcrumb>(
+        "fast-breadcrumb"
+    );
+
+    return { element, connect, disconnect };
+}
+
+describe("Breadcrumb", () => {
```

Comment:
Do you have any tests with regard to breadcrumb's relationship with breadcrumbitems? like stuff around the slot for the items and what if they put something incorrectly in that slot and so on?

Reply:
added one for the separator